sdks/go/pkg/beam/testing/passert/sum.go

```diff
@@ -22,9 +22,9 @@ import (
 	"github.com/apache/beam/sdks/go/pkg/beam/internal/errors"
 )
 
-// Sum validates that the incoming PCollection<int> is a singleton
-// with the given value. Specialized version of Equals that avoids
-// a lot of machinery for testing.
+// Sum validates that the sum and count of elements in the incoming PCollection<int> is
+// the same as the given sum and count, under coder equality. Specialized version of Equals
+// that avoids a lot of machinery for testing.
```

Comment:
Can we also change the last sentence to start: "Sum is a specialized version ...". GoDocs and comments should be full sentences.

Reply:
Noted and reflected changes accordingly.